### ISSUE TITLE
checkra1n 0.11.0 hangs or crashes on userland boot

### ISSUE THREAD

**Issue by ghost** [checkra1n/BugTracker#1689]
**Tell us about your setup:**
What iDevice are you using?
A: iPad 5th Generation (WiFi) Board Config: j71tap to be exact
On what version of iOS is it?
iPadOS 14.0 (18A373)
What version of checkra1n are you using?
Latest 0.11.0
What is your host system (OS version? Hackintosh? VM? etc.)?
An old 2010 white MacBook on Catalina 10.16.5, same error on my HP Laptop running the latest version of Ubuntu 20.04 LTS.
How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)?
Original Apple USB-A cable, tried every USB 3/2 ports. Aswell tried using different cables.
**What are the steps to reproduce the issue?**
1. Connect the iPad to any usb port
2. Run checkra1n as usual
3. When checkra1n says "Booting" the device shows the checkra1n logo then it reboots to a normal apple logo.
...

**What do you expect, and what is happening instead?**

I expect it to continue showing the checkra1n logo until the device has reached SpringBoard, Instead it reboots and the Loader doesnt get installed and I usually get 2 errors from this.

1. usbmuxd (Error -79)
2. Bootstrap Timeout (Error -20)

I have tried to jailbreak with the safe mode option on but the same thing happends. Everything worked fine on 13.7 and below.

**Does the issue also occur if you tick "Safe Mode" in the checkra1n options?**

Yes

**Any other info, error logs, screenshots, ...?**

![image](https://user-images.githubusercontent.com/70339581/93936744-eb826f80-fd26-11ea-852c-2caf3a1009da.jpeg)

^ Same thing happen when I use the CLI version.

**Comment by Catta1997** [comment]
Same thing with same device (iPad 5 wifi), noticed that the reboot happen after` L2TP domain init complete` but I don't think that is related because sometimes crash on `IOMFB use_scaled_brightness: doit=0`.  I have tried a "clean iOS install" but with no success

**Comment by Siguza** [comment]
Can reproduce.

**Comment by noob698** [comment]
I also noticed the same thing happening on old iPadOS 14.0 beta with iPad (5th Generation Wifi)

**Comment by jeffry2berdugo** [comment]
Same here :(
on the same device

**Comment by Epicvoid986** [comment]
Yea, ipad 5th generation (cellular), tried on Mac and Windows with checkn1x, can reproduce, both error code -20 and -79

**Comment by noob698** [comment]
I guess this is happening on all iPad 5th generation

**Comment by shoyrock** [comment]
I have the same problem it was working fine on 13.7 once I tried version 11 everything stopped working

**Comment by leofe4** [comment]
![6A6A4BC3-B0E0-41B8-96DF-EDF504C9476D_1_201_a](https://user-images.githubusercontent.com/22140593/94305921-9f5a4980-ff37-11ea-9249-a0e449006346.jpeg)

I have the same thing happening to me on my ipad 5.

**Comment by mmm286** [comment]
Same problem

**Comment by noob698** [comment]
Hopefully this gets fixed soon. Everyone who are using iPad 5 will experience this crappy problem

**Comment by BorisYeltsin** [comment]
> Hopefully this gets fixed soon. Everyone who are using iPad 5 will experience this crappy problem

and iPad Pro 1st gen

**Comment by Sandwhich253989** [comment]
Same  problem here I have tried 20 times with Linux windows mac on my ipad 5th gen

**Comment by clove7802** [comment]
same problem with my ipad 5th gen on ios14 and ios14.0.1.
hopefully this gets fixed soon.

**Comment by michaelmantz** [comment]
Same here, ipad 5th generation cellular, on my iphone 6s works fine on ios 14.01

**Comment by jazzmaxx** [comment]
Any idea when this will be fixed? Closing the issues doesn‘t fix it. I understand that duplicate tickets need to be closed but also this issue should be worked on.

**Comment by trn1ty** [comment]
Glad to hear I'm not the only one with this issue, and that it's known to the developers.

**Comment by TOTALMICHEL** [comment]
> Any idea when this will be fixed? Closing the issues doesn‘t fix it. I understand that duplicate tickets need to be closed but also this issue should be worked on.

pretty sure that they are working on it, but its a FREE project. they don't get payed to do this, so even it takes a year or more to fix, we should be glad that they took some of their valuable time to give this awesome tool to us.

**Comment by Medlk** [comment]
I have the same issue here using ipad pro 9.7 inch ios 14.0.1... It worked perfectly with ios 13.5.1 but now it's not I v tried it for 30 times in the past 3 days (I change the cable.. Used the method were u unplug the cable before booting ect.. until I given up.. i thought the problem with my iPad now its clear to me.. I hope it fix soon my son is killing me with his noise lol.. Thanks for this great tool developers

**Comment by Vitteliustin** [comment]
Same problem here with ipad pro 9.7 :( hope there will be a fix soon...I've tried macOS Linux Bootrain CLI mode, nothing works...

**Comment by noob698** [comment]
Yeah same with iPad 5th

**Comment by iosfabian** [comment]
Same error on iPhone X iOS 13.7

Tried all given solution options

Anything else I can try?

**Comment by noob698** [comment]
I don’t know. After this problem gets fixed, your iPhone problem might also be fixed. The checkrain before 11.0 got released worked on my iPad when it was running on 13.7. Just turn untested thing on and see if that works

**Comment by Sandwhich253989** [comment]
> I don’t know. After this problem gets fixed, your iPhone problem might also be fixed. The checkrain before 11.0 got released worked on my iPad when it was running on 13.7. Just turn untested thing on and see if that works
What version?

**Comment by Sandwhich253989** [comment]
> I don’t know. After this problem gets fixed, your iPhone problem might also be fixed. The checkrain before 11.0 got released worked on my iPad when it was running on 13.7. Just turn untested thing on and see if that works

What Checkra1n version? What device model?what version iOS?

**Comment by iosfabian** [comment]
@noob698 tried all this simple things
No solution

**Comment by victorcrane** [comment]
iPad Pro  12.9 First Gen ios14 
After jailbreak error -20 stuck on CR app booting screen 
and weird part is when I connect to the ifunbox its says jailed

**Comment by noob698** [comment]
@victorcrane does the checkrain application there though?

**Comment by victorcrane** [comment]
> @victorcrane does the checkrain application there though?

no CR app in springboard

**Comment by noob698** [comment]
Shit

**Comment by iosfabian** [comment]
Is there any update expected in the nearly future?

**Comment by Ismael903** [comment]
Hi i got the same problem error code 79 in the ipad pro 9.7 with ios 14 and use the last checkra1n, does anyone know the solution? I think we have to wait for the developers to work on it, I would like to support them with some donation or some incentive since they work not for profit and we benefit from it.

**Comment by modaifallah** [comment]
Saaaame here, tried ubuntu, mac, ra1nux, terminal and gui, gives checkm8 logo and text, then panics and reboot midway. Ipad 5th generation Wifi.

**Comment by noob698** [comment]
So many participants. Must be a major problem then

**Comment by Medlk** [comment]
Omg I m still trying every solution I see daily with no results except a headache.Did anyone tried the Rainusb version... Plz if anyone find something replay

**Comment by Ismael903** [comment]
I'm trying the same thing every day, and I can't find the solution.  We will have to wait, at least in ipad pro 9.7 first generation with a9x chip does not work, the usbmux error 79 jumps let's wait for it to be solved soon ... greetings, let's not lose hope

**Comment by abacus0101** [comment]
I thought i did something wrong.mhopefully next release of CK fixes this otherwise no jailbreak

**Comment by abacus0101** [comment]
These guys that work on this devote alot of time on it. The only issue I have is if they are going to release,a new jailbreak for 14.0 ect and then say it works on original ipad pro’s, it means no one really tested it fully before releasing it. Thats the only criticism I can see. Now we are all at the mercy of the checkra1n developers. It is after all a beta and they have not updated it yet. Hopefully the next version will add the checkra1n loader app to the devices and we will get a full,jailbreak

**Comment by ShawnTimez** [comment]
To be honest checkra1n beta 0.11.0 just seems broken.

I dont think we are waiting on anything particular for ipad 5th gen. Its running the supported processor. The exploit is there.

I tried running beta checkra1n 0.11.0 when it first came out when my device was still on ios 13.7

The results were checkra1n beta 0.11.0 never would successfully jailbreak the device not even on ios 13.7 however checkra1n 10.2 worked everytime.

I'm convinced there is a small error that is causing a huge issue in the new version of checkra1n and I tried it on multiple devices (beta 0.11.0) and it never worked on anything I tried it on old or new.

Verdict checkra1n 0.11.0 was rushed and untested completely. Im surprised anyone has gotten it to work on anything because i have not.

Also would like to reference this post.
https://github.com/checkra1n/BugTracker/issues/1744

Could it be related? Or are ipad 5th gen users just screwed here?

**Comment by Medlk** [comment]
@ShawnTimez I am no expert in jailbreaking.. But I wanna give this information it may help.. Whenever I tried to jailbreak using 0.11.0 it shows always the error 79 wmand when I use version 0.10.2 it always give me error 77 (for iPad pro 9.7 inch ios 14.0.1 via bootrain) I changed cable/pc always the same problem

**Comment by witsadmin** [comment]
Same thing happening with my iPod 6th Gen. However, 0.10.2 works perfectly

**Comment by Tmarkr** [comment]
is it the A9 chip??? is anyone having this issue trying to jailbreak iPhone 6,6S or the 1st gen SE???

**Comment by hermes814** [comment]
sane problem here, iPad 5th on ios 14.0.1

**Comment by Vitteliustin** [comment]
Still no update! :( ? Hope that someone's working on it and our problem isn't ignored by the Chrckra1n guys...

**Comment by isaamrgt** [comment]
Hi , i found it today , try this https://youtu.be/XlQpZebt-OI

**Comment by Siguza** [comment]
Yes, we are working on the issue. And it's not something you can fix on your end, it has to do with our early boot setup.  
Unfortunately we'll have to go through a lot of code and test on many device and iOS combos to make sure we don't break even more, so it won't be a quick fix. :/

**Comment by shoyrock** [comment]
> Yes, we are working on the issue. And it's not something you can fix on your end, it has to do with our early boot setup.
> Unfortunately we'll have to go through a lot of code and test on many device and iOS combos to make sure we don't break even more, so it won't be a quick fix. :/

Thanks for the update. I was wandering if you guys would be able to post/releases a beta/Canary version so users who brick there Devices trying to use version 11 to jailbreak they are devices are are able to downgrade back to a previous version we use an SHSHBLOB, I was on 13.6.1 when I tried jailbreak he’s in the latest version of Checkra1n it end up breaking my device that’s why I had to update to the last version of iOS, if you guy having problem getting the update out without breaking other things I understand I don’t know how to code So I have no leg to stand on when it comes to that but at least remove the version that are currently not working off the main website so that users gonna stop working their device by attempting to jailbreak with that version

**Comment by TOTALMICHEL** [comment]
For me, checkra1n 11 works on iPhones.
It just fails on iPads. (Even old iPads like mini 2 on 12.4.8)

**Comment by Medlk** [comment]
@Siguza thank u for the replay bro, you're doing a great job

**Comment by hermes814** [comment]
> Hi , i found it today , try this https://youtu.be/XlQpZebt-OI

not working bro

**Comment by hermes814** [comment]
> Yes, we are working on the issue. And it's not something you can fix on your end, it has to do with our early boot setup.
> Unfortunately we'll have to go through a lot of code and test on many device and iOS combos to make sure we don't break even more, so it won't be a quick fix. :/

thanks bro a lot of encouragement for the whole checkra1n team

**Comment by Sandwhich253989** [comment]
> Yes, we are working on the issue. And it's not something you can fix on your end, it has to do with our early boot setup.
> Unfortunately we'll have to go through a lot of code and test on many device and iOS combos to make sure we don't break even more, so it won't be a quick fix. :/

Thanks for the update 😁 @Siguza

**Comment by x766** [comment]
Important! Code :-79 Cause analysis

A user whose code: -78 and-79 could not jailbreak appears，A conflict caused by the wrong installation of a plug-in or a file is almost certain, or is there some other hardware problem?
Take a look at Settings----privacy----analytics and improvements----analytics data----panic-XXXX-XXXX-XX-XX-XXXXXX.ips
my question：
After I swiped the Checkra1n code to about 80%, the iPhone started automatically，The [you may attempt to enter no-substrate mode by holding vol-up during boot(starting with Apple logo until boot completes).] approach doesn't solve my problem.Since I did not install Cydia but rather installed the software via SSH, even the no-Substrate did not solve my problem.
Library\LaunchDaemons This folder contains the plist file installed by me using FILZA software via jailbreak,such as com.matchstic.reprovisiond.plist,It will associate the usr\bin\reprovisiond file, As a result of installing these two files，After I swiped the Checkra1n code to about 80%, the iPhone started automatically，Then the jailbreak failed.Of course,Brush into the checkRA1n code after I choose safe Mode also failed.
Please.Can Lucas or your team create a new pattern?Bootstrap jailbreak bootstrap without loading all files in the Library\LaunchDaemons folder. After success, SSH channel port 44 ROOT will be granted automatically.Users can write the command script from Workflow (Shortcuts) to uninstall the files themselves, or install iTerminal software to uninstall the files via SSH channel port 44. That's the only way I know so far.
In addition,ReProvision, installed via SSH channel port 44 (not cydia, but deb parsed packaged IPA files), had no unusual problems for the first week.In the meantime, I've rebooted the iPhone and then jailbroken it with Checkra1n, and everything seems to be fine, with no unusual problems.All of a sudden one day, I don't remember how the iPhone was rebooted, the first time I booted the jailbreak through Checkra1n, the desktop still had the checkRA1n icon, just like now, the SSH channel was closed, and I forgot to click systeam inside the checkra1n icon to clear the jailbreak environment, but that's later.Boot the jailbreak desktop with checkra1n on the second reboot and the checkra1n icon disappeared, and I was unable to install Cydia, nor was There a Succession plug-in in Cydia to flatten the system.
In September 2020, I heard that Apple added a SEP security processor cache control, which caused problems in the connection between CPU and peripherals. I wonder if my problem is related. 

Tell us about your setup:
What iDevice are you using?
iphone8 D20AP
On what version of iOS is it?
13.6.1
What version of checkra1n are you using?
ubuntu，mac，ra1nux，terminal，gui，CLI，
checkra1nLinux 0.10.0-0.11.0 GUI and CLI，when I entered DFU mode and plugged in USB cable, the Iphone would start up automatically, so I did not try.
Ra1nMacUSB is based on the Mac environment usb drive system.0.10.1-0.11.0 GUI and CLI，when I entered DFU mode and plugged in USB cable, the Iphone would start up automatically, so I did not try.
checkn1x-1.1.2 0.10.1-0.11.0.
What is your host system (OS version? Hackintosh? VM? etc.)?
Ubuntu, Windows and usb sticks,Usb flash drive system based on Mac environment.
I have change the different configuration of Windows Intel CPU start by U disk.
i7-3770k
E3V1231
i5-2300
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)?
Orginal charging cable.
What are the steps to reproduce the issue?
1.
2.
3.
...
What do you expect, and what is happening instead?
Checkra1n Jailbreak successfully opened SSH channel port 44, and Then I used the Linux command to remove the conflicting files over SSH channel port 44.
An SSH channel is actually not open.
Does the issue also occur if you tick "Safe Mode" in the checkra1n options?
Yes, it still happens!
Any other info, error logs, screenshots, ...?
I suspect this has something to do with Apple fixing the LPE bug in 08Tc3wBB. When 08Tc3wBB indicated that Apple had fixed part of it, Apple added a SEP security processor cache control, causing problems in the connection between CPU and peripherals.

The following is my network disk link, because the network environment is special, so in the form of ZIP packaging released to the forum, 
please Siguza watch, thank you.
My English is not good, I hope you can understand my description.

https://310.lanzous.com/iMyTJheqzhe

**Comment by x766** [comment]
@Siguza 
I can't upload the package file, I can't upload the picture, I don't know what's going on?

**Comment by Sandwhich253989** [comment]
What wa s weird was I could open ssh tunnel in 3utools even though it told not jailbroken

**Comment by x766** [comment]
If you have an SSH channel open, you can simply go to the relevant system folder and delete the conflicting files.
![Conflict code](https://user-images.githubusercontent.com/72794154/96265189-b997bc80-0ff7-11eb-9150-190cd0aa7e5d.JPG)
![Uploading Collapse analysis.png…]()

**Comment by x766** [comment]
This is my second picture.
![Uploading Collapse analysis.png…]()

**Comment by OneWorldOneSky** [comment]
Guys! Any luck. Ipad 5th generation 14.0.1 checkra1n 0.11 is not working and I saw developer's command,it's taking time. Anyone find any solution or shortcuts for A9 ipad 5th generation. Let me know please. Thank you in advance

**Comment by BillKG** [comment]
Had the same error but I manage to jailbreak it through cli mode. Ran both cli and gui side by side, gui crashed but cli continued.
Steps:
1. open checkra1n cli
     - run command `./checkra1n -c  `
     - "Waiting for DFU devices" will appear on terminal
2. open checkra1n gui app
    - put device in dfu mode 
    - proceed with normal jail break 
    - once the exploit begins on gui it will most likely crash but cli will continue and jailbreak the device.

macOS Catalina: 10.15.7
checkra1n: Latest 0.11.0
Cable type: Original Apple USB-A cable

<img width="548" alt="Screenshot 2020-10-18 at 10 15 47 AM" src="https://user-images.githubusercontent.com/21240283/96354441-e5678f00-112a-11eb-9845-33d0fd78b2f9.png">

**Comment by CarlKath** [comment]
Did you successful jailbreak an iPhone or an iPad with checkra1n?

**Comment by ShawnTimez** [comment]
> Had the same error but I manage to jailbreak it through cli mode. Ran both cli and gui side by side, gui crashed but cli continued.
> Steps:
> 
> 1. open checkra1n cli
>    
>    * run command `./checkra1n -c  `
>    * "Waiting for DFU devices" will appear on terminal
> 2. open checkra1n gui app
>    
>    * put device in dfu mode
>    * proceed with normal jail break
>    * once the exploit begins on gui it will most likely crash but cli will continue and jailbreak the device.
> 
> macOS Catalina: 10.15.7
> checkra1n: Latest 0.11.0
> Cable type: Original Apple USB-A cable
> 
> <img alt="Screenshot 2020-10-18 at 10 15 47 AM" width="548" src="https://user-images.githubusercontent.com/21240283/96354441-e5678f00-112a-11eb-9845-33d0fd78b2f9.png">

I'm trying this but its still not working. Running on ubuntu Linux both terminals in root

**Comment by BillKG** [comment]
> Did you successful jailbreak an iPhone or an iPad with checkra1n?

iPad

**Comment by CarlKath** [comment]
> > Did you successful jailbreak an iPhone or an iPad with checkra1n?
> 
> 
> 
> iPad

Thanks for the answer, will try your workaround later at home. :)

**Comment by BillKG** [comment]
> > Had the same error but I manage to jailbreak it through cli mode. Ran both cli and gui side by side, gui crashed but cli continued.
> > Steps:
> > 
> > 1. open checkra1n cli
> >    
> >    * run command `./checkra1n -c  `
> >    * "Waiting for DFU devices" will appear on terminal
> > 2. open checkra1n gui app
> >    
> >    * put device in dfu mode
> >    * proceed with normal jail break
> >    * once the exploit begins on gui it will most likely crash but cli will continue and jailbreak the device.
> > 
> > macOS Catalina: 10.15.7
> > checkra1n: Latest 0.11.0
> > Cable type: Original Apple USB-A cable
> > <img alt="Screenshot 2020-10-18 at 10 15 47 AM" width="548" src="https://user-images.githubusercontent.com/21240283/96354441-e5678f00-112a-11eb-9845-33d0fd78b2f9.png">
> 
> I'm trying this but its still not working. Running on ubuntu Linux both terminals in root

Have you tired checkra1n [0.10.2](https://checkra.in/releases/0.10.2-beta)? [Heard](https://twitter.com/SoNick_14/status/1314630957459275776?s=20) it is the more stable version.

**Comment by CarlKath** [comment]
I thought it doesn't work with iOS 14

**Comment by Waze0613** [comment]
Hey folks, @Siguza said this is not something we can fix on our end. We should just trust him and patiently wait for a fix.

**Comment by ShawnTimez** [comment]
10.2 doesnt work for ios 14.

So what exactly did you get success jailbreaking? An ipad on what version of ios with what version of checkra1n?

I tried this on a ipad 5th gen running IOS 14.0.1 and it doesn't work running cli and gui next to eachother. Tried multiple times and with both terminals in root.

> > > Had the same error but I manage to jailbreak it through cli mode. Ran both cli and gui side by side, gui crashed but cli continued.
> > > Steps:
> > > 
> > > 1. open checkra1n cli
> > >    
> > >    * run command `./checkra1n -c  `
> > >    * "Waiting for DFU devices" will appear on terminal
> > > 2. open checkra1n gui app
> > >    
> > >    * put device in dfu mode
> > >    * proceed with normal jail break
> > >    * once the exploit begins on gui it will most likely crash but cli will continue and jailbreak the device.
> > > 
> > > macOS Catalina: 10.15.7
> > > checkra1n: Latest 0.11.0
> > > Cable type: Original Apple USB-A cable
> > > <img alt="Screenshot 2020-10-18 at 10 15 47 AM" width="548" src="https://user-images.githubusercontent.com/21240283/96354441-e5678f00-112a-11eb-9845-33d0fd78b2f9.png">
> > 
> > 
> > I'm trying this but its still not working. Running on ubuntu Linux both terminals in root
> 
> Have you tired checkra1n [0.10.2](https://checkra.in/releases/0.10.2-beta)? [Heard](https://twitter.com/SoNick_14/status/1314630957459275776?s=20) it is the more stable version.

**Comment by nourinou** [comment]
I have the same issue on my ipad 5 i cant get to jailbreak it using peppermint linux on checkra1n 0.11.0 nor the checkn1x both on CLI or Gui 
It just does feels like the system panics on the payload after the apple logo apears its sudden restarts animation and resets to normal boot
Im not qualified to know what even is Happening but all i know that in verbose boot it dont seem like i used to jailbreak
Something is missing

**Comment by itsTyrion** [comment]
> Yes, we are working on the issue. And it's not something you can fix on your end, it has to do with our early boot setup.
> Unfortunately we'll have to go through a lot of code and test on many device and iOS combos to make sure we don't break even more, so it won't be a quick fix. :/

I wonder why all of sudden it's happening to me.. It was working perfectly,

Edit: 1 restore (root fs from the loader app) and retry later, it spits out the same errors but everything worked fine O_o)
![image](https://user-images.githubusercontent.com/17730120/96494317-7b440c80-1246-11eb-8b77-fc297089ca64.png)
![image](https://user-images.githubusercontent.com/17730120/96494702-03c2ad00-1247-11eb-9ce3-c426935dd6e2.png)

**Comment by Waze0613** [comment]
> > Yes, we are working on the issue. And it's not something you can fix on your end, it has to do with our early boot setup.
> > Unfortunately we'll have to go through a lot of code and test on many device and iOS combos to make sure we don't break even more, so it won't be a quick fix. :/
> 
> I wonder why all of sudden it's happening to me.. It was working perfectly,
> 
> Edit: 1 restore (root fs from the loader app) and retry later, it spits out the same errors but everything worked fine O_o)
> ![image](https://user-images.githubusercontent.com/17730120/96494317-7b440c80-1246-11eb-8b77-fc297089ca64.png)
> ![image](https://user-images.githubusercontent.com/17730120/96494702-03c2ad00-1247-11eb-9ce3-c426935dd6e2.png)

Which iOS version ? iPhone or iPad ?

**Comment by Vitteliustin** [comment]
Any update on the situation?? (IPad Pro 9.7 IpadOS 14.01)

**Comment by noob698** [comment]
Nope

**Comment by Waze0613** [comment]
If only checkra1n team didn't say iPad Pro 1st gen on iOS14 was compatible with 0.11.0, we would all have staid on 13.X ....

**Comment by TOTALMICHEL** [comment]
That was, and is the only thing im upset, 
It still says That it works on the website, it should had been edited to say that it does not work on ios 14

https://twitter.com/qwertyoruiopz says that new checkra1n is about to be released :D

**Comment by itsTyrion** [comment]
> > > Yes, we are working on the issue. And it's not something you can fix on your end, it has to do with our early boot setup.
> 
> > > Unfortunately we'll have to go through a lot of code and test on many device and iOS combos to make sure we don't break even more, so it won't be a quick fix. :/
> 
> > 
> 
> > I wonder why all of sudden it's happening to me.. It was working perfectly,
> 
> > 
> 
> > Edit: 1 restore (root fs from the loader app) and retry later, it spits out the same errors but everything worked fine O_o)
> 
> > ![image](https://user-images.githubusercontent.com/17730120/96494317-7b440c80-1246-11eb-8b77-fc297089ca64.png)
> 
> > ![image](https://user-images.githubusercontent.com/17730120/96494702-03c2ad00-1247-11eb-9ce3-c426935dd6e2.png)
> 
> 
> 
> Which iOS version ? iPhone or iPad ?

iPhone 6s (n71mAP board), iOS 13.3

**Comment by Waze0613** [comment]
> That was, and is the only thing im upset, It still says That it works on the website, it should had been edited to say that it does not work on ios 14 https://twitter.com/qwertyoruiopz says that new checkra1n is about to be released :D

Nothing says the new version will include the fix for this bug.

**Comment by TOTALMICHEL** [comment]
It does say that it works on iPhone 7 iOS 14.0.1
Só since i7 is A10, im guessing it should work on A9X.

**Comment by Waze0613** [comment]
Well, this bug seem very specific to A9X. So unless they decided to implement the fix in the upcoming release, we can definitely  imagine a new checkra1n version without it but with A10 support. Wait and see ...

**Comment by abacus0101** [comment]
I would hope the developers read these comments and would fix this bug in the next version. These guys are really good and im sure they will iron this out and get it fixed, fingers crossed

**Comment by Sandwhich253989** [comment]
> I would hope the developers read these comments and would fix this bug in the next version. These guys are really good and im sure they will iron this out and get it fixed, fingers crossed

Yeah

**Comment by Vitteliustin** [comment]
Anyone knows when the next Checkra1n will be launched?  ( The guys have made a great work until now) hope you fix rhe A9X problem with the next update. I've lost my apple accout and reseted my ipad pro 9.7 and now is just sitting there waiting for your resolve.

**Comment by TOTALMICHEL** [comment]
Also waiting for the jailbreak for iPad Pro 1 gen.
Updated it because it said it was supported.
Now I can’t use dolphin to play games during my 1h20m daily travel to work.

Guess they will release it after telling Apple all they need to fix on A16 (too late to fix A15) kind of wish they keep quiet about A14 bugs.

**Comment by fennectech** [comment]
Same  waiting for a fix here.   Internet said i could jailbreak on 14.1 but nope  iPad Pro 1st gen

**Comment by ShawnTimez** [comment]
If anyone is curious I just tried the patched version of checkra1n 0.11.0 beta that supposedly works for iPhone 7-10 on ios 14


Well it doesn't work for this issue. Still panics and boots like normal unjailbroken
Ipad 5th gen ios 14.0.1

**Comment by carloslopweb** [comment]
Yo estoy igual he probado también con iPad Pro 1 y no hace jalibreak, habrá que esperar a que checkra1n saque la versión 0.11.1, esto es solo un parche por eso no se ha reparado el error......

**Comment by Vitteliustin** [comment]
Yes,  still not working on ipad pro 9.7 ios 14 :(  still it panics on boot.....I just want an onest answer is someone working on this? Or I should sell my ipad for  parts ....

**Comment by WakaBlazer** [comment]
Now I regret updating my iPad Pro A9X I was using it properly with iOS 13.7 untethered now it’s just like a device for parts... Checkra1n please next time do test every device before releasing an update now we are stuck 😔😔😔😔 and stop giving false information and fake news

**Comment by TOTALMICHEL** [comment]
Maybe next time don’t use stolen devices...

**Comment by TOTALMICHEL** [comment]
> Now I regret updating my iPad Pro A9X I was using it properly with iOS 13.7 untethered now it’s just like a device for parts... Checkra1n please next time do test every device before releasing an update now we are stuck 😔😔😔😔 and stop giving false information and fake news

Maybe next time don’t use stolen devices...

no checkra1n dev will ever help people use stolen devices. so just be glad if someday it works again.

**Comment by Waze0613** [comment]
> > Now I regret updating my iPad Pro A9X I was using it properly with iOS 13.7 untethered now it’s just like a device for parts... Checkra1n please next time do test every device before releasing an update now we are stuck 😔😔😔😔 and stop giving false information and fake news
> 
> Maybe next time don’t use stolen devices...
> 
> no checkra1n dev will help people use stolen devices. so just be glad if someday it works again.

You should not jump to such conclusions. Mine, for instance, is a lost device. Its owner never claimed it. So I was happy to be able to use it until I updated because checkra1n team announced it was compatible with version 0.11.0 ...

**Comment by raph2705** [comment]
Good for you, but nonetheless it is a beta version, you cannot expect it to work nor to be tested for every device on the (lost devices) market ;-) and it works like a charm with a (lost) XS max in 14.1 so there is no fake news here.

**Comment by TOTALMICHEL** [comment]
It works on a XS MAX !?
A12? 😵

**Comment by itsTyrion** [comment]
> It works on a XS MAX !?
> 
> A12? 😵

No?? Never did and never will

**Comment by raph2705** [comment]
Sorry for the confusion it's an X, the XS is the one I just received.

**Comment by Sandwhich253989** [comment]
> If anyone is curious I just tried the patched version of checkra1n 0.11.0 beta that supposedly works for iPhone 7-10 on ios 14
> 
> Well it doesn't work for this issue. Still panics and boots like normal unjailbroken
> Ipad 5th gen ios 14.0.1

U are right the unofficial patched version of Checkra1n does not work on my device ipad 5 iPadOS 14.0.1

**Comment by TOTALMICHEL** [comment]
It doesn’t work because it only disables sep on A10 and A11.
We needed to add the code for disabling on A9X
I took a look at the patch but is on that unreable code 😩

Guess I will have to wait a bit more to get dolphin working again on my long trip to work

**Comment by Sandwhich253989** [comment]
Oh ok I understood

**Comment by TOTALMICHEL** [comment]
The very bad downside of the patch (and that is why checkra1n didn’t it)
It that with sep off there is no passcode, no fáceid no Touch ID.
If you try to set one the device panics, so unless you are willing to use a open device, that patch is not for you

With that said, in my case I wouldn’t mind since I just wanna play GameCube and wii games on my way to work using dolphin
That device does not need to have any private stuff on it.

**Comment by noob698** [comment]
Ok

**Comment by CarlKath** [comment]
> The very bad downside of the patch (and that is why checkra1n didn’t it)
> 
> It that with sep off there is no passcode, no fáceid no Touch ID.
> 
> If you try to set one the device panics, so unless you are willing to use a open device, that patch is not for you
> 
> 
> 
> With that said, in my case I wouldn’t mind since I just wanna play GameCube and wii games on my way to work using dolphin
> 
> That device does not need to have any private stuff on it.

Ist there some possibility to get my hands on that patch?
I don't use any Passcode either.

**Comment by Medlk** [comment]
Hi members is there any progress abt the bug for iPad!?

**Comment by carloslopweb** [comment]
Hola, yo de momento no he conseguido nada me sigue fallando hacer jalibreak en iPad Pro 1genr el parche no soluciona el error 79 habrá que esperar más adelante haber si checkra1n saca su herramienta.....

**Comment by abdellahbouras** [comment]
iPad 5 wifi+cellulariPadOS 14.1 (18A8395checkra1n 
 0.11.0 beta mackbook 2015 hight sir 10.13.6  resultat /Bootstrap Timeout (Error -20)
ubuntu.usbmuxd (Error -79)

**Comment by fennectech** [comment]
Anyone got a source for the patch?

**Comment by Sandwhich253989** [comment]
I recommend u not to use it but if u want then go to Twitter @exploitguy3d

**Comment by Vitteliustin** [comment]
Well let's hope that the official release of the new checkra1n will be different from the pach and will fix the A9X issue, i know that there are a lot of users that need that jailbreak :) GO CHECKRA1N TEAM!

**Comment by TOTALMICHEL** [comment]
The patch is included in pingo os 2.3.3 
So it checkrain 0.12 uses that pongo version 
A9X will be fixed

**Comment by shoyrock** [comment]
can you post a link or screenshot

**Comment by TOTALMICHEL** [comment]
Check the source code.


Synopsys_otg.c (its the usb stack - remember what error a9x gives? ;) )
It fixes a interrupt asserting before dma transfer is complete on some devices.

I already compiled 2.3.3 and managed to boot to pongoos on my A9X i evem got to use some pongoos commands using pongoterm

Can’t boot checkra1n cause i don’t have access to checkra1n-kpf2-12.0,14.0 module

even tried using checkrain with -ck (pongo 2.3.3.bin) but it dosent gives acess to modules inside checkra1n.
(https://user-images.githubusercontent.com/20833921/98221120-26dba500-1f47-11eb-87ee-e89e4cdd109f.png)

**Comment by nxahjs** [comment]
This happens on my iPhone X, which is pretty odd

**Comment by Tmarkr** [comment]
I see that the newest checkra1n(0.12.0) is out now, but I don't see anything about A9(x) compatibility, especially on the iPad ....fingers crossed

**Comment by Galactic-Dev** [comment]
checkra1n 0.12.0 fixes this on my A9X iPad

**Comment by ghost** [comment]
This has now been fixed in checkra1n 0.12.0!🥳

**Comment by Siguza** [comment]
Marking this as fixed. 🎉

**Comment by Medlk** [comment]
At least i can end this day at a good note thanks to all u checkra1n team

**Comment by trn1ty** [comment]
So... I'm still getting the same issue. Let me repeat my specs/setup/etc:
**Laptop OS: Arch Linux x86_64 latest
Laptop hardware: Thinkpad T420, iPhone plugged into a USB A 2.0 port
iOS 14.2 on iPhone SE 2016
checkra1n beta 0.12.0**
Trying again: Launching as user, entering recovery mode. checkra1n has failed to detect my phone successfully entering recovery both of two times tested. Fortunately when checkra1n is launched as the root user (and only as the root user) it can detect that the iPhone has been put into recovery mode, if it can't detect an iPhone entering recovery in user mode then that's probably the cause of the previous error. Entering DFU mode OK, boots as normal, **but then displays USB error -76** (see screenshot):

![2020-11-08-232236_1600x900_scrot](https://user-images.githubusercontent.com/26193059/98500139-9f798480-2219-11eb-8c20-bbb835a4afe3.png)

Then the phone doesn't get a checkra1n or Cydia app on the home menu and I doubt the SSH server is up (let me check... yup, connection refused). It looks like the bug, or at least my bug #1693 which was flagged as a duplicate of this one, has not been fixed :(

-- That being said, thank you very much for the update anyway because it's still very useful to me for getting my device out of recovery mode. Got stuck in that and it took me a couple hours on the r/jailbreak_ Discord server back before it was taken off-line to figure out how to exit using some arcane irecovery commands I've since forgotten!

**Comment by noob698** [comment]
Yeah same with iPad 5th generation but mine says “all done” but still has no checkrain app

**Comment by noob698** [comment]
Wait safe mode fixes the problem. Thank you devs!

**Comment by jeffry2berdugo** [comment]
Checkrain 0.12.0 Solved my problem on iPad 5th Generation IOS 14, thank you Devs! Ur the best.

**Comment by amakiller** [comment]
What iOS ? 14.1 or 14,2 ) iPad 5 Gen

**Comment by jeffry2berdugo** [comment]
> What iOS ? 14.1 or 14,2 ) iPad 5 Gen

14.0 Don't know if it work on least version

**Comment by amakiller** [comment]
I did jailbreak iPad 5 Gen with IOS 14.1.
But I am little bit scared to try to update to 14.2 before someone else did.

**Comment by michaelmantz** [comment]
Just jaibreaked iPad 5th gen on updated ios 14.2

**Comment by jeffry2berdugo** [comment]
> Just jaibreaked iPad 5th gen on updated ios 14.2

You're god bro, thank u!!!

**Comment by michaelmantz** [comment]
The only problem I had was on my iphone 6s, it was jailbroken on 14.1 and I wasn't able to update over the air to ios 14.2, after I updated through itunes, I wasn't able to jailbreak, I had to restore the phone

**Comment by iosfabian** [comment]
Still the same problem on my device (iPhone X iOS13.7)

**Comment by trn1ty** [comment]
Tried again, got an error -63. Tried to boot into safe mode, once again got an error -76.

**Comment by deroyabun** [comment]
Works great on iOS 14.2 with iPad Pro 1st 9,7“ gen.

**Comment by Sandwhich253989** [comment]
Works with iPad 5 ipadOS 14.2 .

**Comment by trn1ty** [comment]
@Siguza issue may need to be re-opened, tagging you in case you haven't seen this (sorry for the spam if you have).

**Comment by mind-overflow** [comment]
Yep, totally still happening.
Ubuntu 20.10 on kernel 5.8.28, checkra1n 0.12.0, iPhone 6 on iOS 12.4.
The phone, sometimes, actually reboots in jailbroken state; some other times it just hangs. Still, the error is shown every time.

**Comment by x766** [comment]
Hey, guys, please upload some useful pictures to solve the code-78 and code-79 problem.

Take a look at Settings----privacy----analytics and improvements----analytics data----panic-XXXX-XXXX-XX-XX-XXXXXX.ips

When uploading your pictures here, be careful to hide the key,key to protect your privacy. Thank you.

my question：
IOS usually calls /Library/LaunchDaemons (less commonly used /Library/LaunchAgents) at startup; / System/Library/LaunchDaemons(System services  here is generally put)
If installed Cydia Substrate, then starts can also through plist definition and calls, call the load/Library/MobileSubstrate/DynamicLibraries dylib program inside.

The first 2(3) are system-level startup calls, while Cydia Substrate's can only take effect after the environment of Cydia Substrate is installed. Besides, holding down the "volume +" key at startup can completely skip plug-in calls based on it, but system-level calls cannot be skipped! So if it's a plug-in issue based on the Cydia Substrate environment, you can press "volume +" to skip it, but if it's a system-level startup call, it's a problem that gets stuck!

Since I did not install Cydia but rather installed the software via SSH,even the no-Substrate did not solve my problem.

Library\LaunchDaemons This folder contains the plist file installed by me using FILZA software via jailbreak,such as com.matchstic.reprovisiond.plist,It will associate the usr\bin\reprovisiond file, As a result of installing these two files，After I swiped the Checkra1n code to about 80%, the iPhone started automatically，Then the jailbreak failed.Of course,Brush into the checkRA1n code after I choose safe Mode also failed.

In general, if you have the OPENSSH service installed and you can start it, you can handle the basic issues (you can execute terminal commands, modify permissions, and so on), but now you can't start the SSH service.

This issue is mainly about fixing the jailbroken preset APT environment, where checkra1n files may lose execution in the /var/lib directory or fail to place files there.

My guess is that this solution has a very good chance of solving the code-79 problem, just my guess.

Tell us about your setup:
What iDevice are you using?
iphone8 D20AP
On what version of iOS is it?
13.6.1

My English is very poor, I hope you can understand my description.

This picture prompts me:Something went wrong in the background in front of launch.
![Uploading Conflict code.JPG…]()

**Comment by x766** [comment]
![Conflict code](https://user-images.githubusercontent.com/72794154/98975014-8a219480-2550-11eb-8346-1ba3a8ec5cd9.JPG)

**Comment by x766** [comment]
@Siguza 

 @nullpixel 

 @jacc
